docs/advanced_topics/api/v3/usage.md

```
@@ -0,0 +1,620 @@
+(api_v3_custom_page_fields)=
+
+### Custom page fields in the API
+
+Wagtail sites contain many page types, each with their own set of
+fields. The `pages` endpoint will only expose the common fields by
+default (such as `title` and `slug`).
+
+For anonymous callers, the `?type` parameter selects a specific page
+model and unlocks the fields it has listed in `api_fields`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)